selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -18,9 +19,11 @@ def __init__(self, dbc_name, CP, VM):
     self.graMsgStartFramePrev = 0
     self.graMsgBusCounterPrev = 0
 
+    self.use_lanelines = Params().get('EndToEndToggle') != b'1'
```

Comment:
this is a temporary change in openpilot UI's to make it clear what mode you're in. it doesn't make sense to carry this over to the car interface

Reply:
@adeebshihadeh is there another way we can know if we should expect lanelines to be present? There's a difference in car HUD expectations.

If we shouldn't count on this param, I think we should drop the 'lane line visible' signal at all times, and just always show a green indicator when Openpilot is active, and grey when it isn't.

Reply:
let's just forget the laneless case and always go with the "With lanes (lane-specific states)" case